save-cli/src/nativeMain/kotlin/org/cqfn/save/cli/SaveConfigReader.kt

```diff
@@ -53,11 +61,27 @@ fun SaveProperties.getFields() = this.toString().dropWhile { it != '(' }.drop(1)
 @Suppress("TOO_LONG_FUNCTION")
 fun createConfigFromArgs(args: Array<String>): SaveProperties {
     // getting configuration from command-line arguments
+    if (args.isEmpty()) {
+        throwErrorAndExitNotFoundDir()
+    }
     val configFromCli = SaveProperties(args)
     tryToUpdateDebugLevel(configFromCli)
     logDebug("Properties after parsed command line args:\n${configFromCli.getFields()}")
     // reading configuration from the properties file
-    val propertiesFile = (configFromCli.testRootPath ?: defaultConfig().testRootPath) + DIRECTORY_SEPARATOR + "save.properties"
+    val testFiles = configFromCli.testFiles
+    try {
+        if (testFiles != null && testFiles.isNotEmpty()) {
+            FileSystem.SYSTEM.metadata(testFiles[0].toPath())
+        }
+    } catch (e: FileNotFoundException) {
+        throwErrorAndExitNotValidDir(testFiles!!.get(0).toPath())
+    }
+    val testRootPath = if (testFiles == null || testFiles.isEmpty() || !FileSystem.SYSTEM.metadata(testFiles!!.get(0).toPath()).isDirectory) {
```

Comment:
Why get(0)

Reply:
i do not like logic on the exceptions. Does okio has any method like `exists` for Path

Reply:
Fixed